py/torch_tensorrt/dynamo/conversion/converter_registry.py

```diff
@@ -305,23 +305,32 @@ def unique_targets(self):
         """Returns the set of unique converter targets stored across all registries"""
         return set.union(*[set(registry.keys()) for registry in self.registries])
 
+    # TODO: Make this a static method since it does not need state
     def qualified_name_or_str(self, target: Target) -> str:
         """Returns string representation of an FX Node target"""
         if isinstance(target, str):
             return target
         else:
             return _get_qualified_name(target)
 
-    def display_all_available_converters(self) -> str:
-        """Returns a string with all converters and their source, separated by newlines"""
-        available_converters = "Available converters in ATen registries with counts:\n"
-
+    def get_converter_support_info(self) -> Dict[str, Dict[str, str]]:
+        """Returns a dictionary of targets backed by at least one converter"""
+        available_converters = {}
         for target in sorted(
             self.unique_targets(), key=lambda target: self.qualified_name_or_str(target)
         ):
             _, registry_data = self.get_all_converters_with_target(
                 target, return_registry_info=True
             )
```

Comment:
The `registry_data` dictionary returned by `get_all_converters_with_target` is `Dict[str, int]`, mapping the registry name to the number of instances of converters for that target in each registry. The function header for the output should be updated to `Dict[str, Dict[str, int]]`